src/App.js

```diff
@@ -46,8 +49,14 @@ class App extends React.Component {
 
 // component render
   render() {
-    console.log(this.state);
-    if(this.state.movies.length === 0) {
+    if (this.state.error !== "") {
+      return(
+        <div>
+          <p>{this.state.error} Error</p>
+          <p>Sorry, something's gone wrong. Please try again.</p>
+        </div>
+      )
+    } else if (this.state.movies.length === 0) {
```

Comment:
we should take away the strictly equals and replace it with !this.state.movies.length